index2.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+   
+    <link rel="stylesheet" href="styles2.css">
+</head>
+<body>
+    <div id="smallScreen">
+        <span><a href=""></a>Welcome</span>
+        <span><a href=""><i class="fa fa-align-justify"></i></a></span>
+    </div>
+    <div id="main">
+        <div class="navigation">
+            <nav><ul>
+                <li><a href="#">Portoflio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul></nav>
+        </div>
+        <div class="headline">
+            <p class="header">
+                <h2>Hey there. I’m Franck sefu.<br>
+                <span>I’m a software developer</span></h2>
+            </p>
+            <p class="content">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div class="icon">
+                <a href="https://www.facebook.com/franck.sefu/">
+                  <i class="fab fa-facebook-f"></i>
+                </a>
+                <a href="https://twitter.com/franck_sefu">
+                  <i class="fab fa-twitter"></i>
+                </a>
+                <a href="https://www.linkedin.com/in/franck-sefu-884705254/">
+                  <i class="fab fa-linkedin-in"></i>
+                </a>
+                <a href="https://github.com/francksefu">
+                    <i class="fab fa-github"></i>
+                  </a>
+            </div>
+            <p class="action">
+                <button><img src="Disabled.png" id="img"></button>
```

Comment:
- [ ] This image is not showing up. I think the source link is not correct. kindly consider fixing it.
![ew](https://user-images.githubusercontent.com/96683843/212146287-5fa44fc7-7548-46a4-8b7f-2dfe991390ed.jpg)

Reply:
- [ ] I think this image is not showing because we do not have it anywhere in your files, make sure you have pushed it on your GitHub repo and see what will happen

<img width="343" alt="1" src="https://user-images.githubusercontent.com/77843050/212207389-9c7c5345-4fe9-47a3-b261-0c0443a7d31c.png">